### ISSUE TITLE
test failing when defining transient variables in solidity

### ISSUE THREAD

**Issue by gabririgo** [sc-forks/solidity-coverage#897]
When running solidity coverage using transient storage in solidity (supported by solc v0.8.28) I get the following error:

```
Error in plugin solidity-coverage: Error: Could not instrument: protocol/extensions/adapters/AUniswapRouter.sol. (Please verify solc can compile this file without errors.) extraneous input 'tokensInCount' expecting {';', '='} (44:30)
```
Error happens here:
https://github.com/RigoBlock/v3-contracts/blob/9a0274b93fa9f0ca0046350227afb866e810782f/contracts/protocol/extensions/adapters/AUniswapRouter.sol#L44

Steps to reproduce:

- run rigoblock v3-contracts repo from feat/uniswap v4 branch, run yarn coverage
- alternatively, define a transient storage variable in a project, the error will be returned.

**Comment by cgewecke** [comment]
@gabririgo Thanks for reporting. I think we just need to update the parser dependency to latest here to fix this...will publish a patch doing that in the next couple of hours. 

If you want to try this change locally you can coerce your version of solidity-parser/parser to `0.19.0` using [yarn resolutions][1].

Transient support was added to the parser in this release:
https://github.com/solidity-parser/parser/releases/tag/v0.19.0

[1]: https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/#toc-how-to-use-it

**Comment by cgewecke** [comment]
Should be fixed in `0.8.14` (just published). Closing, but please lmk if you're still seeing problems.

**Comment by gabririgo** [comment]
works like a charm with 0.8.14! thank you
